### PR TITLE
EVM example

### PR DESCRIPTION
**Pull request by mothran**
Added an example project for running EVM transactions within the zkvm using the [revm](https://crates.io/crates/revm) crate.

### CODE REVIEW

File: examples/Cargo.toml [hunk 1]
```diff
@@ -2,6 +2,7 @@
 members = [
     "chess",
     "digital-signature",
+    "evm",
```
[the remark this answers is not in the document]
Comment (mothran) [reply examples/Cargo.toml:5]: I was basing that off the `waldo` example which uses sub-crates to mimic a workspace, example here:
https://github.com/risc0/risc0/blob/parker/evm-example/examples/waldo/Cargo.toml#L14

[the remark this answers is not in the document]
Comment (flaub) [reply examples/Cargo.toml:5]: I'm surprised that `cargo-sort` didn't cover `evm/example`. I wonder, if we add it, will it pick it up?

[the remark this answers is not in the document]
Comment (mothran) [reply examples/Cargo.toml:7]: Turns out cargo-sort does not explore `path = ..` dependencies of sub-crates. So we have to add all the sub-crates to the top level workspace def if we want cargo-sort to automatically pick them up.

**Review by flaub — APPROVED** [review]

**Review by tzerrell — APPROVED** [review]
💯

**Pull request merged**